content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/enabling-github-actions-with-minio-storage.md

```diff
@@ -11,23 +11,25 @@ topics:
   - Infrastructure
   - Storage
 redirect_from:
-  - /admin/github-actions/enabling-github-actions-with-minio-gateway-for-nas-storage
-shortTitle: MinIO Gateway for NAS storage
+  - /admin/github-actions/enabling-github-actions-with-minio-storage
```

Comment:
👋🏻 again @aj-jester, we're coordinating to get you a full review 🔜 but I wanted to leave this one comment while I'm here.

These are redirects for past URLs, so we need to leave the existing entry unchanged, and add another one since we're renaming the article here.

```suggestion
  - /admin/github-actions/enabling-github-actions-with-minio-gateway-for-nas-storage
  - /admin/github-actions/enabling-github-actions-for-github-enterprise-server/enabling-github-actions-with-minio-gateway-for-nas-storage
```

Reply:
Thanks @lecoursen I accepted the commit suggestion. Will wait for the full review!